tests/system/small/ml/test_preprocessing.py

```diff
@@ -580,8 +597,12 @@ def test_one_hot_encoder_different_data(penguins_df_default_index, new_penguins_
                 [{"index": 2, "value": 1.0}],
             ],
         },
-        dtype=pd.ArrowDtype(
-            pa.list_(pa.struct([("index", pa.int64()), ("value", pa.float64())]))
+        dtype=(
+            pd.ArrowDtype(
```

Comment:
seems to be a repetitive pattern, might be worth abstracting away in a method `get_compatible_dtype` inside `tests.system.utils`, or even `PANDAS_VERSIONS`

Reply:
In this case the dtype is exactly the same. No need for a function. I just created a constant.